dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
If someone removed `OutputStream` and added `outputStream` or `output_stream` (which won't happen during the time we keep those redirects) it wouldn't be too big an issue.